nncf/torch/model_analyzer.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+import torch
+from torch import nn
+from torch.quantization.fake_quantize import FakeQuantize
+
+from nncf.common.graph.graph import NNCFNode
+from nncf.torch.graph.operator_metatypes import OPERATORS_FUSED_METATYPES
+from nncf.torch.graph.operator_metatypes import OPERATORS_WITH_BIAS_METATYPES
+from nncf.torch.module_operations import UpdateWeight
+from nncf.torch.nncf_network import NNCFNetwork
+from nncf.torch.quantization.layers import BaseQuantizer
+
+
+def get_potential_fused_node(node_name: str, model: NNCFNetwork) -> Optional[NNCFNode]:
+    """
+    Get next node that can contain fused bias in runtime.
+
+    :param node_name: The node name.
+    :param model: The model that contains this operation.
+
+    :return: The node that can be fused or None.
+    """
+    graph = model.nncf.get_original_graph()
+    target_node = graph.get_node_by_name(node_name)
+
+    if target_node.metatype in OPERATORS_WITH_BIAS_METATYPES:
+        next_nodes = graph.get_next_nodes(target_node)
+        for node in next_nodes:
+            if node.metatype in OPERATORS_FUSED_METATYPES:
+                return node
+    return None
+
+
+def is_node_with_fused_bias(node: NNCFNode, model: NNCFNetwork) -> bool:
+    """
+    Checks if the node has a fused bias.
+
+    :param node: The node to check.
+    :param model: The model that contains this operation.
+    :return: Return `True` if `node` corresponds to the operation
+        with bias (bias is added to the output tensor of that operation),
+        `False` otherwise.
+    """
+    fused_node = get_potential_fused_node(node.node_name, model)
+    node_module = model.get_containing_module(node.node_name)
```

Comment:
This call generate the following warning:
```
/home/susloval/work/projects/nncf/nncf/torch/nncf_network.py:934: FutureWarning: Old style of accessing NNCF-specific attributes and methods on NNCFNetwork objects is deprecated. Access the NNCF-specific attrs through the NNCFInterface, which is set up as an `nncf` attribute on the compressed model object.
For instance, instead of `compressed_model.get_graph()` you should now write `compressed_model.nncf.get_graph()`.
The old style will be removed after NNCF v2.5.0
```
Please migrate to the new NNCFNetwork representation here and elsewhere. cc' @vshampor

Reply:
Fixed